### PR TITLE
Mothroach Operatives

### PR DESCRIPTION
**Pull request by DarkenedSynergy**
## About the PR

Added a nuclear operative mothroach to the syndicate uplink, with fitting reinforcement radio.
## Why / Balance

As spoken about in https://discord.com/channels/968983104247185448/1213723732183613480! Adds more options to the "allies" section of the uplink. Similar to the mouse operative all be it with slightly different drawbacks(increased fire damage, bigger hitbox) and benefits (faster movement without gravity, possibility to eat head clothing, being a mothroach).
## Technical details

Mainly yml.
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
https://github.com/user-attachments/assets/caf69aa5-b397-4aca-be36-fb70bc6f3fb4
mothperatives
https://streamable.com/60yind

## Breaking changes

**Changelog**

:cl:
add: Added mothroach operative reinforcements to the uplink to work alongside their cheesy brethren!

### CODE REVIEW

**Comment by DarkenedSynergy** [comment]
Had the same issue as the mouse reinforcement teleporter having the wrong parent with the recent upstream merge.

File: Resources/Prototypes/Datasets/Names/mothroach_death_commando.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Datasets/Names/mothroach_death_commando.yml:1]: Delta-V folder please

File: Resources/Locale/en-US/deltav/ghost/roles/ghost-role-component.ftl [hunk 1]
```diff
@@ -2,7 +2,12 @@ ghost-role-information-nukie-mouse-name = Nuclear Operative Mouse
 ghost-role-information-nukie-mouse-description = A Nuclear Operative reinforcement for the Syndicate.
 ghost-role-information-nukie-mouse-rules = Normal syndicate antagonist rules apply. Work with whoever called you in, and don't harm them.
                                            The crew is allowed to kill you without warning.
-                                           You are allowed to attack the crew and destroy the station without provocation. 					
+                                           You are allowed to attack the crew and destroy the station without provocation.
```
Comment (DangerRevolution) [inline Resources/Locale/en-US/deltav/ghost/roles/ghost-role-component.ftl:5]:
```suggestion
                                           You are allowed to attack the crew and destroy the station without provocation.

```

Leave a line for better readability

File: Resources/Locale/en-US/deltav/ghost/roles/ghost-role-component.ftl [hunk 2]
```diff
@@ -24,7 +29,7 @@ ghost-role-information-recruiter-rules =
     You are just a recruiter so do not act like a full-on antagonist, i.e. no killing people.
 
 ghost-role-information-silvia-name = Silvia
-ghost-role-information-silvia-description = You are Silvia the space snake and the CMO's charming companion.    
+ghost-role-information-silvia-description = You are Silvia the space snake and the CMO's charming companion.
```
Comment (DangerRevolution) [inline Resources/Locale/en-US/deltav/ghost/roles/ghost-role-component.ftl:32]: why is this touched?

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/nukiemothroach.yml [hunk 3]
```diff
@@ -0,0 +1,168 @@
+﻿- type: entity
+  name: nuclear operative mothroach
+  parent: MobCockroach
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/nukiemothroach.yml:3]: Why is this parented to Cockroach?

You can cut down the components to like... three or four by not parenting it to the cockroach

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/nukiemouse.yml [hunk 4]
```diff
@@ -154,9 +154,6 @@
     channels:
     - Syndicate
   # make the player a traitor once its taken
-  - type: AutoTraitor
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/nukiemouse.yml:157]: Why are you changing this?

File: Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml [hunk 5]
```diff
@@ -90,6 +90,7 @@
   - type: EmitSoundOnUse
     sound: /Audio/Animals/cat_meow.ogg
 
+
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml:93]: ?

File: Resources/Prototypes/Datasets/Names/mothroach_death_commando.yml
Comment (MilonPL) [inline Resources/Prototypes/Datasets/Names/mothroach_death_commando.yml:1]: Should be in the DeltaV folder.

File: Resources/Textures/Mobs/Animals/mothroach.rsi/meta.json
Comment (DangerRevolution) [inline Resources/Textures/Mobs/Animals/mothroach.rsi/meta.json:1]: Fix this.

File: Resources/Textures/DeltaV/Objects/Devices/communication.rsi/plushie-radio.png
Comment (DangerRevolution) [inline Resources/Textures/DeltaV/Objects/Devices/communication.rsi/plushie-radio.png:1]: This doesn't really fit in line with the cheese radio mouse operatives use; plus this is extremely valid

File: Resources/Textures/DeltaV/Objects/Devices/communication.rsi/meta.json [hunk 6]
```diff
@@ -1,15 +1,18 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Inspired by the cheese wedge sprite from tg station https://github.com/tgstation/tgstation/blob/master/icons/obj/food/food.dmi modified by @ps3moira",
-  "size": {
+  "copyright": "Cheese radio inspired by the cheese wedge sprite from tg station https://github.com/tgstation/tgstation/blob/master/icons/obj/food/food.dmi modified by @ps3moira. plushie-radio sprited by jimbobbythe19th. ",
+    "size": {
```
Comment (DangerRevolution) [inline Resources/Textures/DeltaV/Objects/Devices/communication.rsi/meta.json:5]: ?

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Comment by TadJohnson00** [comment]
Will require further discussion but, ideally, we'll be moving away from animal reinforcements in future (i.e. syndicats and mouse ops). Hopefully we'll have a consensus by the time this PR is up for review.

**Comment by TadJohnson00** [comment]
@DarkenedSynergy is this PR still being worked on?

**Comment by DarkenedSynergy** [comment]
> @DarkenedSynergy is this PR still being worked on?

Ill probably get to it at somepoint, not an urgent thing to PR. But yes I plan on finishing this.

**Comment by MilonPL** [comment]
Heya! Since this PR has gotten no commits or activity in the past 3 months, I'm going to close it. If you ever come back to this, feel free to open a new PR.

**Pull request closed without merging**